includes/KSES.php

```diff
@@ -54,20 +54,7 @@ public function register() {
 			return;
 		}
 
-		// The third param of the below filter was only added in WP 5.4.1.
-		if ( version_compare( get_bloginfo( 'version' ), '5.4.1', '>=' ) ) {
-			add_filter( 'wp_insert_post_data', [ $this, 'filter_insert_post_data' ], 10, 3 );
-
-			return;
-		}
-
-		// Fallback for WP < 5.4.1. Affects all post types, not just the web-story one.
-		// TODO: Remove once support for WP 5.3 is dropped.
-		add_filter( 'safe_style_css', [ $this, 'filter_safe_style_css' ] );
```

Comment:
Should any of these methods be fully removed, too? Or are these still used elsewhere? I'm seeing other usages of some of these filters but are these still necessary?

Also, seeing this comment below in this file for `safecss_filter_attr`:
`Use safe_style_disallowed_chars filter once WP 5.5+ is required.`

Should this method be replaced? Or is this done in another PR?

Reply:
+1 to removing unused methods.

As for safe_style_disallowed_chars, it should be done in a separate ticket, but I don‘t think it actually makes a lot of sense maintenance wise

Reply:
`filter_safe_style_css` is used in `filter_insert_post_data`. Why would be remove it?

> As for safe_style_disallowed_chars, it should be done in a separate ticket, but I don‘t think it actually makes a lot of sense maintenance wise

Agree, this is for another ticket.

Reply:
> Agree, this is for another ticket.

Should we create one? Or remove the comment if it won't be necessary?

Reply:
Follow up issue. https://github.com/google/web-stories-wp/issues/7992